src/common/io_priority.cc

```diff
@@ -12,14 +12,17 @@
  *
  */
 
-#include "io_priority.h"
-
 #include <unistd.h>
+#if defined(__FreeBSD__)
```

Comment:
@joscollin 
I've been using `#if defined()` for all my submissions. 
This will allow me an easy grep to find all typical FreeBSD stuff.
And in the whole code there is a mismatch of all the possible forms.